Dockerfile

```diff
@@ -1,3 +1,14 @@
+# Build step for check-payload tool
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.13-2.1729776560 as check-payload-build
+
+WORKDIR /opt/app-root/src
+
+ARG CHECK_PAYLOAD_VERSION=0.3.2
+
+RUN curl -k -s -L -o check-payload.tar.gz "https://github.com/openshift/check-payload/archive/refs/tags/${CHECK_PAYLOAD_VERSION}.tar.gz" && \
```

Comment:
I do not think that `curl -k` should be used to download data from GitHub.  `-k` means `--insecure`.

Reply:
This is being fixed via #349.